src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml

```diff
@@ -140,55 +140,54 @@
                         }
                     </td>
                 </tr>
+            </tbody>
+        </table>
+
+        <h2 class="heading-medium">
+            Apprenticeship details
+        </h2>
+        <table class="details-list details-list-new">
+            <tbody>
                 <tr>
                     <th scope="row">Apprenticeship training course </th>
                     <td>@Model.Data.TrainingName </td>
                 </tr>
-                @*<tr>
-                        <th>End-point assessor </th>
-                        <td>@Model.Data.EndPointAssessor</td>
-                    </tr>*@
                 <tr>
-                    <th scope="row">Training start date </th>
+                    <th scope="row">Planned training start date</th>
                     <td>
                         @if (@Model.Data.StartDate.HasValue)
                         {
-                            @Model.Data.StartDate.Value.ToGdsFormat()
+                            @Model.Data.StartDate.Value.ToGdsFormatWithoutDay()
```

Comment:
Does this return the full month IE  February instead of Feb?

Reply:
Yes it does...

![image](https://user-images.githubusercontent.com/12608907/60097149-b31e6b00-974a-11e9-9338-7efbb4b7b08e.png)

Reply:
Cool no worries, I had used ToGdsFormatLongMonthNameWithoutDay() in my stories so wasn't sure  if there's a difference as the name definitely suggests one 😂